src/Export/Search/ProductSearcher.php

```diff
@@ -34,6 +34,7 @@ protected function fetchProducts(
         ?string $productId = null
     ): ProductCollection {
         $criteria = $this->buildCriteria($limit, $offset, $productId);
+        $criteria->addAssociation('visibilities');
```

Comment:
There is an `PRODUCT_ASSOCIATIONS` constant within the library, please add it there